src/seraphis_main/CMakeLists.txt

```
@@ -59,6 +60,7 @@ monero_add_library(seraphis_main
 
 target_link_libraries(seraphis_main
   PUBLIC
+    async
     cncrypto
```

```suggestion
  PUBLIC
    cncrypto
```
See comment on `enote_finding_context.h`.